crengine/src/lvrend.cpp

```diff
@@ -3805,7 +3793,7 @@ int pagebreakhelper(ldomNode *enode,int width)
 
 // Prototypes of the 2 alternative block rendering recursive functions
 int  renderBlockElementLegacy(LVRendPageContext & context, ldomNode * enode, int x, int y, int width , int usable_right_overflow);
-void renderBlockElementEnhanced( FlowState * flow, ldomNode * enode, int x, int width, int flags );
+void renderBlockElementEnhanced(FlowState * flow, ldomNode * enode, int x, int width, lUInt32 flags );
```

Comment:
My space ! My balance ! :)

Reply:
I don't know why this happened, maybe it's an IDE, they are too smart now :)